Gemfile.lock

```diff
@@ -71,73 +71,73 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (2.0.0)
       rails (>= 7.0)
-    activejob (7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activejob (7.0.4)
+      activesupport (= 7.0.4)
       globalid (>= 0.3.6)
-    activemodel (7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activerecord (7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activestorage (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activemodel (7.0.4)
+      activesupport (= 7.0.4)
+    activerecord (7.0.4)
+      activemodel (= 7.0.4)
+      activesupport (= 7.0.4)
+    activestorage (7.0.4)
+      actionpack (= 7.0.4)
+      activejob (= 7.0.4)
+      activerecord (= 7.0.4)
+      activesupport (= 7.0.4)
       marcel (~> 1.0)
       mini_mime (>= 1.1.0)
-    activesupport (7.0.2.3)
+    activesupport (7.0.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    acts_as_tenant (0.5.1)
+    acts_as_tenant (0.5.2)
       rails (>= 5.2)
       request_store (>= 1.0.5)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.4.0)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    airbrussh (1.4.1)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.2.0)
       activerecord (>= 3.2, < 8.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.7.0)
+    apipie-rails (0.8.2)
       actionpack (>= 5.0)
       activesupport (>= 5.0)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.573.0)
-    aws-sdk-core (3.130.0)
+    aws-partitions (1.632.0)
+    aws-sdk-core (3.150.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.525.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-kms (1.55.0)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-kms (1.58.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.113.0)
+    aws-sdk-s3 (1.114.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.4)
-    aws-sigv4 (1.4.0)
+    aws-sigv4 (1.5.1)
       aws-eventstream (~> 1, >= 1.0.2)
-    bcrypt (3.1.17)
+    bcrypt (3.1.18)
     better_errors (2.9.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
     bindex (0.8.1)
     binding_of_caller (1.0.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.11.1)
+    bootsnap (1.13.0)
       msgpack (~> 1.2)
     builder (3.2.4)
-    capistrano (3.17.0)
+    capistrano (3.17.1)
```

Comment:
### [capistrano](https://capistranorb.com/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.17.0 → 3.17.1

[_[change-log](https://github.com/capistrano/capistrano/releases), [source-code](https://github.com/capistrano/capistrano), [gem-diff](https://my.diffend.io/gems/capistrano/3.17.0/3.17.1)_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/capistrano/capistrano/compare/v3.17.0...v3.17.1).

These are the individual commits:
- (cc4f31f) [Install newer SSH client to fix CI for old Rubies (#2106)](https://github.com/capistrano/capistrano/commit/cc4f31fdfcb4a21c569422a95868d0bb52844c75)
- (15104dc) [Remove reference to the deprecated :scm variable (#2111)](https://github.com/capistrano/capistrano/commit/15104dc4bfe92c546dba54bbbdf32f026ee18d8d)
- (ec6b61b) [Switch cucumber tests to use https instead of git:// (#2112)](https://github.com/capistrano/capistrano/commit/ec6b61b694ac6264028327c5570f9de4a91acd18)
- (9e331e3) [Release 3.17.1](https://github.com/capistrano/capistrano/commit/9e331e319745731708ceaac6eecb559701bb9c48)


</details>